training_scripts/gsgnn_dt/distill.yaml

```diff
@@ -0,0 +1,20 @@
+---
+version: 1.0
+gsf:
+  basic:
+    backend: gloo
```

Comment:
i think it's better to use nccl in this case.

Reply:
nccl is faster than gloo. currently, we can only use gloo if the model has DGL's sparse learnable embeddings. For BERT distillation, we should use nccl to make the training faster.

Reply:
I just tried nccl but it got stuck.

Reply:
Please add a TODO to enable nccl.
NCCL is more efficient for Bert training.

Reply:
added